opm/autodiff/BlackoilModelParameters.cpp

```diff
@@ -46,6 +46,7 @@ namespace Opm
         tolerance_mb_    = param.getDefault("tolerance_mb", tolerance_mb_);
         tolerance_cnv_   = param.getDefault("tolerance_cnv", tolerance_cnv_);
         tolerance_wells_ = param.getDefault("tolerance_wells", tolerance_wells_ );
+        solve_wellEq_initially_ = param.getDefault("solve_wellEq_initially",solve_wellEq_initially_);
```

Comment:
All other parameters have `lower_case_names` and I think this should, too (for consistency in user-facing options).